Workflow.podspec

```diff
@@ -13,7 +13,7 @@ Pod::Spec.new do |s|
   s.cocoapods_version = '>= 1.7.0'
 
   s.swift_versions = ['5.0']
-  s.ios.deployment_target = '11.0'
+  s.ios.deployment_target = '12.0'
```

Comment:
Minor:  Should probably bump the package.swift target version as well.

Reply:
This will get reverted before the PR opens for review; it was to get around Cocoapods erroring locally on integration.

Reply:
👍